BootloaderCommonPkg/Include/Library/FirmwareUpdateLib.h

```diff
@@ -38,6 +39,15 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 
 #define CAPSULE_FLAGS_CFG_DATA  BIT0
 
+#define FIRMWARE_UPDATE_STATUS_SIG       SIGNATURE_32 ('F', 'S', 'T', 'S')
```

Comment:
Modified FIRMWARE_UPDATE_STATUS_SIG to "FWUS".

These two are different structures and different tables, One is in the reserved region and other table will go in to ACPI table. Two different signatures are required to avoid confusion

Reply:
For reserved region, I think it is OK to use same signature used in ACPI table. But it is also fine to define a new one, I means I prefer using FIRMWARE_UPDATE_STATUS_SIGNATURE instead of FIRMWARE_UPDATE_STATUS_SIG.

Reply:
Done